astropy/time/formats.py

```diff
@@ -837,7 +856,7 @@ class TimePlotDate(TimeFromEpoch):
       >>> import matplotlib.pyplot as plt
       >>> jyear = np.linspace(2000, 2001, 20)
       >>> t = Time(jyear, format='jyear', scale='utc')
-      >>> plt.plot_date(t.plot_date, jyear)
+      >>> plt.plot_date(t.plot_date, jyear) # doctest: +IGNORE_OUTPUT
```

Comment:
Why was this needed?

Reply:
because `plot_date` returns a `Line` object which lead to a doctest failure.

Reply:
That makes sense, but makes me wonder why it was passing tests before. Did something else change in the meantime that made this doctest line suddenly start failing?

Reply:
@maxnoe - did this fail for you locally or in the CI tests? (good to change regardless, but also good to understand indeed why this did not trigger problems before!)

Reply:
It failed locally, did not check the CI, I fixed this together with other test failures so I don't know whether this also affected CI or whether something with my local setup was strange. Can change back to check.

Reply:
Actually, right now we instruct the test to be skipped: https://github.com/astropy/astropy/blob/ef40111640f9f93215eea87eb793da9c7093de68/astropy/time/formats.py#L35

If with you `IGNORE_OUTPUT`, this works, perhaps we can change it to a matplotlib requirement?
```
__doctest_requires__ = {'TimePlotDate': ['matplotlib']}
```
(Though if you prefer to just leave things as they were, that's fine too!)

Reply:
I think this is best for another PR, will revert the change here.